### PR TITLE
Init

### PR DESCRIPTION
**Pull request by qadan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+
+## Introduction
+
+Modules and functionality added to Discoverygarden's Institutional Repository
+implementation.
+
+**N.B.** Functionality here is branched from the original `dgi_ir` project;
+pieces of that module migrated to here were considered integral to DGI's
+Institutional Repository implementation rather than integral to Research Output
+itself. As such, this module _must be left public_.
```
Comment (adam-vessey) [inline README.md]: Not... sure I understand this assertion, that it must be left public?

File: dgi_ir_customizations.libraries.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+drupal:
+  version: "1.0.0"
+  license:
+    name: GPL
+    url: https://www.gnu.org/licenses/gpl.html
+    gpl-compatible: true
+  dependencies:
+    - core/jquery
+    - core/jquery
```
Comment (adam-vessey) [inline dgi_ir_customizations.libraries.yml]: Dupe?

File: modules/dgi_ir_spreadsheet_ingest_example/dgi_ir_spreadsheet_ingest_example.info.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+name: DGI IR Spreadsheet Ingest Example
+description: Provides a base CSV migration template and script for creating Research Output nodes, based on the out-of-the-box Research Output implementation.
+type: module
+package: DGI
+core: 8.x
+dependencies:
+  - islandora_institutional_repository
+  - islandora_spreadsheet_ingest
```
Comment (adam-vessey) [inline modules/dgi_ir_spreadsheet_ingest_example/dgi_ir_spreadsheet_ingest_example.info.yml]: ... `public` vs `private` stuff... strange to have a public repo pointing at a private requirement.

File: README.md (hunk 1 above)
Comment (qadan) [reply README.md]: just trying to be transparent for the eventual publicization of this, like ... we had a lot of functionality that was baked into our theme stuff, which is being removed from the main module; i don't want to get into a situation where we're just removing things and making them private and not communicating what's going on

Comment (adam-vessey) [reply README.md]: > not communicating

... we're not rebasing and removing things from the history... the PRs are there... should the PRs have more verbose messages perhaps? Or _do_ we intend to rebase and or snapshot the repo and push that? I feel like I'm still missing something here.

File: modules/dgi_ir_spreadsheet_ingest_example/dgi_ir_spreadsheet_ingest_example.info.yml (hunk 3 above)
Comment (adam-vessey) [reply modules/dgi_ir_spreadsheet_ingest_example/dgi_ir_spreadsheet_ingest_example.info.yml]: This is still a thing...

**Review by IAMlKeno — APPROVED** [review]

**Pull request merged**